lib/dynflow/world.rb

```diff
@@ -25,6 +25,7 @@ def initialize(config)
       @connector            = config_for_world.connector
       @middleware           = Middleware::World.new
       @client_dispatcher    = spawn_and_wait(Dispatcher::ClientDispatcher, "client-dispatcher", self)
+      @scheduler            = try_spawn_scheduler(config_for_world)
```

Comment:
We should limit the scheduler to run only on words with executor: the reason: the worlds without executor are usualy the client once, that are not expected to run long-term.